sumpy/expansion/local.py

```diff
@@ -417,21 +410,28 @@ def translate_from(self,
             # Use the axis as the first dimension to vary so that the below
             # algorithm is O(p^{d+1}) for full and O(p^{d}) for compressed
             dims = [axis, *list(range(axis)), *list(range(axis + 1, self.dim))]
+
             # Start with source coefficients. Gets updated after each axis.
-            cur_dim_input_coeffs = src_coeffs
+            cur_dim_input_coeffs = cur_dim_output_coeffs = list(src_coeffs)
```

Comment:
This just gives a default value to `cur_dim_output_coeffs` because `pyright` was complaining about it. Is it confusing?

Reply:
It is confusing to me. Would it instead be possible to manufacture a list of bogus values?